components/tooltip/test/tooltip-truncating.visual-diff.js

```diff
@@ -30,9 +30,11 @@ describe('d2l-tooltip truncating', () => {
 
 		it(testCase.name, async function() {
 			const selector = `#${testCase.name}`;
-			await page.$eval(`${selector}${testCase.focus}`, (content) => {
+			const delay = await page.$eval(`${selector}${testCase.focus}`, (content) => {
 				content.focus();
+				return content.nextElementSibling.delay;
```

Comment:
This is probably better.
```suggestion
				return content.parentNode.querySelector('d2l-tooltip').delay;
```

Reply:
Yeah.  Certainly clearer too!  😄